src/Faker/Provider/en_ZA/Address.php

```diff
@@ -5,10 +5,10 @@
 class Address extends \Faker\Provider\en_US\Address
 {
     protected static $cityPrefix = array('North', 'East', 'West', 'South', 'New', 'Lake', 'Port');
-    protected static $citySuffix = array('fontein','town', 'ton', 'land', 'ville', 'berg', 'burgh', 'borough', 'bury', 'view', 'port', 'mouth', 'stad', 'furt', 'chester', 'mouth', 'fort', 'haven', 'side', 'shire');
+    protected static $citySuffix = array('fontein','town', 'ton', 'land', 'ville', 'berg', 'burgh', 'borough', 'bury', 'view', 'port', 'mouth', 'stad', 'furt', 'chester', 'fort', 'haven', 'side', 'shire');
```

Comment:
Will you run PHP CS fixer later? Short array syntax + space after comma?

Reply:
Can't do short array syntax yet as we still support PHP 5.3, see https://github.com/fzaninotto/Faker/blob/768354c630e9f3644fb1cb3c6d6bef0a1b45d7e9/composer.json#L13.

Reply:
But its outdated and should be dropped in a minor version